### ISSUE TITLE
[Bug?]: Removed storybook.manager.js in v6 is still supported by @storybook/theming v7.1 

### ISSUE THREAD

**Issue by AdrianLsk** [redwoodjs/redwood#9008]
### What's not working?

By removing storybook.manager.js it's not possible to use @storybook/theming according to official [docs](https://storybook.js.org/docs/7.1/react/configure/theming)

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by jtoar** [comment]
Hey @AdrianLsk thanks for reporting! I must've missed this. In their v7 migration guide, Storybook says 

> The Storybook manager is no longer built with Webpack. Now it's built with esbuild. Therefore, it's no longer possible to configure the manager. Esbuild comes preconfigured to handle importing CSS, and images.
>
> Source: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-more-configuration-for-manager


But the docs say otherwise as you've pointed out. I guess they just removed programatic configuration of the manager? Regardless, we should add support for this file back, even in its limited capacity.

**Comment by AdrianLsk** [comment]
Right, that looks confusing, but I think the guide talks about configuring esbuild to handle CSS and other assets instead of using storybook manager for that not that manager itself is not used anymore. :) Especially this line:

`This means the preset value managerWebpack is no longer respected, and should be removed from presets and main.js files.`
